### ISSUE TITLE
Support for unquote fragments

### ISSUE THREAD

**Issue by pallix** [melpon/memoize#2]
Would it be possible to support unquote fragments?

For the moment this does not work:

```
        name = :some_dynamically_computed_name
        defmemo unquote(name)() do
           123
        end

```

**Comment by melpon** [comment]
Thank you for reporting. I will fix it.

**Comment by melpon** [comment]
I fixed at 1.2.8. Please try to latest version.

**Comment by pallix** [comment]
Yes it works. Thanks for your reactivity! I let you close the issue.
